river/main.zig

```diff
@@ -32,8 +32,36 @@ const usage: []const u8 =
     \\
 ;
 
+fn testConfigPath(comptime fmt: []const u8, args: anytype) std.fmt.AllocPrintError!?[:0]const u8 {
+    const path = try std.fmt.allocPrintZ(util.gpa, fmt, args);
+    std.os.access(path, std.os.X_OK) catch {
+        util.gpa.free(path);
+        return null;
+    };
+    return path;
+}
+
+fn getStartupCommand() std.fmt.AllocPrintError!?[:0]const u8 {
+    if (std.os.getenv("XDG_CONFIG_HOME")) |xdg_config_home| {
+        if (try testConfigPath("{}/river/init", .{xdg_config_home})) |path| {
+            return path;
+        }
+    } else if (std.os.getenv("HOME")) |home| {
+        if (try testConfigPath("{}/.config/river/init", .{home})) |path| {
+            return path;
+        }
+    }
+    if (try testConfigPath("/usr/local/etc/river/init", .{})) |path| {
+        return path;
+    }
```

Comment:
I don't see a reason we should look here. Do you have a strong reason why we should do so?
```suggestion
```

Reply:
Convention. "Manually" installed things go to `/usr/local/`, which becomes relevant if we decide to ship a default config (per #146, f.e.).

Reply:
I think manually installing stuff system wide is an antipattern and people should use `-/.local/` for that use case. (I should probably amend the build command example in the readme I realize).

Reply:
The exact system dir is something we may want to make configurable at build time in the future though, I'm sure there's at least one distro out there that doesn't use `/etc/`

Reply:
Sounds reasonable. I'll remove the `/usr/local/` part for now.